net/ola/Makefile

```diff
@@ -64,6 +64,8 @@ HOST_CONFIGURE_ARGS += \
 	--disable-doxygen-html \
 	--disable-doxygen-doc
 
+HOST_CXXFLAGS = -std=c++11
```

Comment:
x86-64 build test fails, you might want to have a look.

Reply:
x86_64 build fail is a CI issue, it is not related to this commit. Almost all PRs fail on x86_64 build due to CI cryptodev issue. you may check another (merged and unmerged) PRs - the same story

Reply:
OK!